data/shaders/basic_vs.glsl

```diff
@@ -1,13 +1,22 @@
 #version 150 core
-#include <locals>
 #include <globals>
 
 in vec4 a_Position;
 in vec4 a_Normal;
 in vec2 a_TexCoord;
 out vec2 v_TexCoord;
+out vec4 v_Color;
+
+in vec4 a_World0;
```

Comment:
how about adopting a different prefix for the instance variables? say, `i_XXX`.

Reply:
I'm not familiar with naming conventions in shaders so yes, will do